stories/Components/MegaMenu/megamenu.scss

```diff
@@ -1,3 +1,5 @@
+@import "../../assets/scss/variables";
```

Comment:
Believe this line needs to go as well (including it here breaks the inhertence order)